app/controllers/browse_controller.rb

```diff
@@ -12,6 +12,12 @@ def index
     response = Magma::Client.instance.query(
       token, params[:project_name], [ :project, '::first', '::identifier' ]
     )
+
+    if response.code != '200'
```

Comment:
Same comment from the previous PR: I'm not sure why this would run, as Magma::Client should raise a client error for 400/500s - if it is running, something is wrong upstream

Reply:
Well we can put a check in the Magma Client. But Timur should always check the return from an external service.